types/index.d.ts

```diff
@@ -5377,6 +5377,243 @@ export interface DocumentNsCommands<ReturnType = unknown> {
   pageSource(
       callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<string>) => void,
   ): Awaitable<IfUnknown<ReturnType, this>, string>;
+
+  /**
+   * Inject a snippet of JavaScript into the page for execution in the context of the currently selected frame. The executed script is assumed to be synchronous.
+   * The script argument defines the script to execute in the form of a function body. The value returned by that function will be returned to the client.
+   *
+   * The function will be invoked with the provided args array and the values may be accessed via the arguments object in the order specified.
+   *
+   * Under the hood, if the `body` param is a function it is converted to a string with `<function>.toString()`. Any references to your current scope are ignored.
+   *
+   * To ensure cross-browser compatibility, the specified function should not be in ES6 format (i.e. `() => {}`).
+   * If the execution of the function fails, the first argument of the callback contains error information.
+   *
+   * @alias executeScript
+   * @alias document.executeScript
+   *
+   * @example
+   * describe('execute script', function() {
+   *   it('executes a script in browser', function(browser) {
+   *     browser.execute(function(imageData: string) {
```

Comment:
```suggestion
   *     browser.document.execute(function(imageData: string) {
```

Reply:
This is the TSDoc for `execute` alias of the command. Are we just keeping the `executeScript` name for the command, or only advertising the `executeScript` in all examples but keeping `execute` alias as well?